exchanges/graphcache/src/store/store.ts

```diff
@@ -70,18 +70,26 @@ export class Store implements Cache {
       const queryType = schema.getQueryType();
       const mutationType = schema.getMutationType();
       const subscriptionType = schema.getSubscriptionType();
+
       if (queryType) queryName = queryType.name;
       if (mutationType) mutationName = mutationType.name;
       if (subscriptionType) subscriptionName = subscriptionType.name;
 
+      this.updates = {
+        [mutationName]: (opts.updates && opts.updates[mutationName]) || {},
```

Comment:
I'm kind of confused by this; we should be doing this normalisation in the traversal logic but internally all root types should have the standard names once they leave the schema; so I'm wondering if this is 100% correct